python/tk_desktop/grouping_list_view.py

```diff
@@ -114,7 +114,7 @@ class GroupingListView(ActionListView):
     expanded_changed = QtCore.Signal(str, bool)
 
     def __init__(self, parent=None):
-        QtGui.QListView.__init__(self, parent)
+        ActionListView.__init__(self, parent)
```

Comment:
Understood. My personal preference is to use `super` because I don't have to go back and change all these lines when/if the base class changes. No biggie. :)